Neptune/IMessage.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Neptune
```

Comment:
Any reason for Neptune?

Reply:
Yes, I've been listening to a lot of Neptune Estate https://www.youtube.com/watch?v=wQf3A2ohX7c

Reply:
Valid reasons: No

Reply:
Sounds like a good reason for me lol